### PR TITLE
Release v2.0.0 beta.1

### PR DESCRIPTION
**Pull request by jdrew82**
This PR is focused on the updates required to support Nautobot 2.0. The changes involved include:

- Updated all imports to be derived from new module locations.
- Updated all models that no longer have slugs to use replacement field.
- Updated navigation to use new NavMenu elements.
- Updated metrics to use new Jobs model attributes.
- Updated example Jobs to use new Location model instead of Region/Site along with updating IPAddress to specify parent Prefix.
- Updated example Jobs to use new Job pattern with passed variables.
- Updated Job loading to use new register_jobs function.
- Updated logging in example Jobs to use new logger on JobResult.
- Updated Infoblox integration to work with Nautobot 2.0.
- Refactored Infoblox integration to have tags applied to imported objects after sync is complete.
- Added pylint-nautobot to dev dependencies.
- Addressed various linter complaints.
- Corrected tests to address field and model changes.

Both example import Job and Infoblox -> Nautobot import Job are functional with test instances.

### CODE REVIEW

**Review by Kircheneer — CHANGES_REQUESTED** [review]
pylint-nautobot needs some configuration to actually be used, see [here](https://docs.nautobot.com/projects/pylint-nautobot/en/latest/getting_started/).

Also I would like to get a chance to review the actual 2.0 changes which at this point are already in the branch, how do we want to handle that?

File: docs/admin/release_notes/version_2.0.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+# v1.3 Release Notes
+
```
[the remark this answers is not in the document]
Comment (jdrew82) [reply docs/admin/release_notes/version_2.0.md]: Good catch! I thought I changed that.

**Comment by jdrew82** [comment]
> pylint-nautobot needs some configuration to actually be used, see [here](https://docs.nautobot.com/projects/pylint-nautobot/en/latest/getting_started/).
> 
> Also I would like to get a chance to review the actual 2.0 changes which at this point are already in the branch, how do we want to handle that?

So all changes are currently in `next-2.0` branch. If we want to instead have that be the base branch for 2.0 development and have a PR for the 2.0 changes in the `next-2.0` branch currently pushed through a PR, I'm happy to rename the branch and do it that way.

**Comment by jdrew82** [comment]
> pylint-nautobot needs some configuration to actually be used, see [here](https://docs.nautobot.com/projects/pylint-nautobot/en/latest/getting_started/).
> 
> Also I would like to get a chance to review the actual 2.0 changes which at this point are already in the branch, how do we want to handle that?

I'm pretty sure I've add the required configuration for pylint-nautobot to the pyproject.toml file as defined in the referenced docs. I've already fixed all of the issues brought up by the plugin.

**Comment by Kircheneer** [comment]
> > pylint-nautobot needs some configuration to actually be used, see [here](https://docs.nautobot.com/projects/pylint-nautobot/en/latest/getting_started/).
> > Also I would like to get a chance to review the actual 2.0 changes which at this point are already in the branch, how do we want to handle that?
> 
> I'm pretty sure I've add the required configuration for pylint-nautobot to the pyproject.toml file as defined in the referenced docs. I've already fixed all of the issues brought up by the plugin.

That is correct, sorry I didn't see it because it is not in the diff :)

**Review by Kircheneer — APPROVED** [review]

**Comment by jdrew82** [comment]
> > > pylint-nautobot needs some configuration to actually be used, see [here](https://docs.nautobot.com/projects/pylint-nautobot/en/latest/getting_started/).
> > > Also I would like to get a chance to review the actual 2.0 changes which at this point are already in the branch, how do we want to handle that?
> > 
> > 
> > I'm pretty sure I've add the required configuration for pylint-nautobot to the pyproject.toml file as defined in the referenced docs. I've already fixed all of the issues brought up by the plugin.
> 
> That is correct, sorry I didn't see it because it is not in the diff :)

That would most likely be because that change was done prior to this PR being opened.

**Comment by jdrew82** [comment]
Closing to rename branches and redo integration of changes for 2.0.

**Pull request closed without merging**